user_docs/changes.t2tconf

```diff
@@ -2,5 +2,11 @@
 
 % Make ticket references into links.
 %!PreProc: "#([0-9]+)" "[#\1 https://github.com/nvaccess/nvda/issues/\1]"
+
+% Make GitHub username references into links.
+% Github username may only contain alphanumeric characters or hyphens.
+% Maximum is 39 characters.
+%!PreProc: ", @([A-z0-9\-]{1,39})" "[@\1 https://github.com/nvaccess/nvda/commits?author=\1]"
```

Comment:
I don't think using `A-z` is what you want as that will also include`[\]^_` which fall between the upper and lower case alphabetic ranges.
I think what you want is `A-Za-z`.

Reply:
```suggestion
%!PreProc: ", @([A-Za-z0-9\-]{1,39})" "[@\1 https://github.com/nvaccess/nvda/commits?author=\1]"
```